### ISSUE TITLE
Consider rewording technical information in F3

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#4660]
During discussions in [response to an existing issue](https://github.com/w3c/wcag/pull/1035#issuecomment-3120228052) it was pointed out by some members of the WCAG 2 Task Force that the second sentence of the description may no longer be accurate, and that the failure technique may contain other technical inaccuracies, which should be addressed.
> The CSS background-image property was designed for decorative purposes and it is not possible to associate text alternatives with images that are included via CSS.

**Comment by patrickhlauke** [comment]
dropping this here from the old issue #80 

> randomly ... F3 has reared its head again in a recent conversation https://eldritch.cafe/@juliemoynat/115355236855876491
>
> would be good if we could try and de-crappify it

**Comment by juliemoynat** [comment]
What I understand from reading the #80 issue is that this part of the third point of the F3 test procedure "and is also available when the CSS image is not displayed" should not be in a technique associated to SC 1.1.1 but 1.3.1.

So as it is associated to SC 1.1.1, this point should be removed from the technique. That's logical. And I think it should be done because the technique would be clearer for everyone.

But there is a question: should we create another technique to address the point which can be an accessibility issue (high contrast mode, users disabling background images)?

There are issues about these questions: #623 and #2889.

And there is the [F24](https://www.w3.org/WAI/WCAG22/Techniques/failures/F24) that says ([pointed in this comment](https://github.com/w3c/wcag/issues/80#issuecomment-548186222)):

> Background color may also be specified using a background image with the CSS property 'background-image' or with the CSS property 'background' (with the URI of the image, e.g., 'background: url("images/bg.gif")'). With background images, it is still necessary to specify a background color, because users may have images turned off in their browser. But the background image and the background color need to be checked. 

I think that if we care about text contrasts while deactivating background images, we should care about alternative texts being visible while deactivating background images.

**Comment by mbgower** [comment]
@juliemoynat 

The point of this new issue was to determine if the quoted statement early in the technique document is _technically_ true, and if the technique contains statements that are similarly challengeable with current technologies.

I assume you are talking specifically about the wording in the latter part of this check (which I have put in bold):
> If an image does convey important information, the information is provided to assistive technologies and is **also available when the CSS image is not displayed**

Looking at that (long) discussion in #80 , I'm not entirely sure why that phrase was left in place, in the end. So if you have the ability, please generate a new PR where that part of the final check is removed. I will ensure that gets brought before the task force. However, I would not consider such a PR to close this issue, which is focused on a technical question.

**Comment by patrickhlauke** [comment]
> Looking at that (long) discussion in https://github.com/w3c/wcag/issues/80 , I'm not entirely sure why that phrase was left in place, in the end. So if you have the ability, please generate a new PR where that part of the final check is removed. I will ensure that gets brought before the task force.

filed #4814

**Comment by patrickhlauke** [comment]
#4814 and #4823 have now been merged. I believe this closes the issue here
